.travis.yml

```diff
@@ -18,7 +18,8 @@ before_install:
     - phpenv config-rm xdebug.ini || return 0
 
 install:
-    - travis_retry composer install
+    - composer --prefer-source install
```

Comment:
Why `--prefer-source` ? This defeats the caching of `$HOME/.composer/cache/files` which persists the composer cache for archives.

Reply:
It was for debugging, will revert.